### DISCUSSION TITLE
Supported Websites

### DISCUSSION THREAD

**Discussion by dcwestra** [karlomikus/bar-assistant#284] [General]
The documentation mentions that drink recipes can be imported from certain websites and that there are some websites already supported. However, I can't find a list of those supported websites. Does this list exist? If so, where can I find it?

**Comment by karlomikus** [comment]
Sites with official support are listed here: https://github.com/karlomikus/bar-assistant/tree/develop/app/Scraper/Sites

For sites without support, BA will still try to extract information

**Reply by dcwestra** [reply]
Thanks! Does bar assistant automatically scrape these sites? Or do I have to tell it to?

**Reply by karlomikus** [reply]
You need to provide an URL with the recipe information, so for example if you give it `https://punchdrink.com/recipes/milano-torino/` it will import the whole recipe information.
